src/renderer/template/codemirror-one-dark.css

```diff
@@ -0,0 +1,92 @@
+/*
+    Name:       one-dark 1.1.1
+    Author:     Török Ádám (http://github.com/Aerobird98)
+    Original Atom One Dark Theme (https://github.com/atom/one-dark-ui & https://github.com/atom/one-dark-syntax)
+*/
+
+/* basic */
+
+/* FIXME: still uses github-light theme as a base */
+
+.cm-s-github-light {
```

Comment:
Notice how it has `github-light` in it's name instead of `one-dark`. I tried everyting but the default theme would always show up so I had to resort to this.